decryption_utils/Go/decrypt.go

```diff
@@ -66,51 +66,58 @@ func decryptFile(privateKey *rsa.PrivateKey, encryptedKey []byte, filename strin
 	decryptedKey, err := rsa.DecryptOAEP(
 		sha256.New(), rand.Reader, privateKey, encryptedKey, []byte(base))
 	if err != nil {
-		panic(err)
+		fmt.Println("Failed to decrypt encrypted key")
+		os.Exit(3)
 	}
-	/* #nosec -- Command line util requires reading a file that is passed via an argument */
-	ciphertext, err := ioutil.ReadFile(filename)
+
+	ciphertext, err := ioutil.ReadFile(filepath.Clean(filename))
 	if err != nil {
-		panic(err)
+		fmt.Println("Failed to read encrypted file")
+		os.Exit(4)
 	}
 
 	var plaintext []byte
 	key := [32]byte{}
 	copy(key[:], decryptedKey[0:32])
 	plaintext, err = decryptCipher(ciphertext, &key)
 	if err != nil {
-		panic(err)
+		fmt.Println("Failed to decrypt file")
+		os.Exit(5)
 	}
 
 	fmt.Printf("%s", plaintext)
 }
 
 func getPrivateKey(loc string) *rsa.PrivateKey {
-	/* #nosec -- Command line util requires reading a file that is passed via an argument */
-	pkFile, err := os.Open(loc)
+
+	pkFile, err := os.Open(filepath.Clean(loc))
 	if err != nil {
-		panic(err)
+		fmt.Println("Failed to open private key")
+		os.Exit(6)
 	}
 
 	pemfileinfo, _ := pkFile.Stat()
-	var size int64 = pemfileinfo.Size()
+	var size = pemfileinfo.Size()
 	pembytes := make([]byte, size)
 	buffer := bufio.NewReader(pkFile)
 
 	_, err = buffer.Read(pembytes)
 	if err != nil {
-		log.Panic(err)
+		fmt.Println("Failed to read private key")
+		os.Exit(7)
 	}
 
 	data, _ := pem.Decode([]byte(pembytes))
 	err = pkFile.Close()
 	if err != nil {
-		log.Panic(err)
+		fmt.Println("Failed to decode private key")
```

Comment:
Should this be a message about `pkFile.Close()`?

Reply:
Good catch.  Fixed now.